<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>830</width>
    <height>480</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>830</width>
    <height>480</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>830</width>
    <height>480</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Profesores</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <widget class="QRadioButton" name="radioButton_codigo">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>10</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Tahoma</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Código</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_nombre">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>10</y>
     <width>82</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Tahoma</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Nombre</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bt_eliminar">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>420</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Verdana</family>
     <pointsize>18</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(7, 70, 124);
font: 18pt &quot;Verdana&quot;;
border-radius: 10px;
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Eliminar</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>811</width>
     <height>351</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Verdana</family>
     <pointsize>11</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QHeaderView::section {
    background-color:  rgb(7, 70, 124);
	color: rgb(255, 255, 255);
	font: 12pt &quot;Verdana&quot;;
    border: 0px solid #ff0000;
    height: 32px;
}

QTableWidget {
	font: 11pt &quot;Verdana&quot;;
    border: 1px solid;
}

QTableWidget::item{
    selection-background-color: rgb(7, 70, 124);
	selection-color: rgb(255, 255, 255);
}
</string>
   </property>
   <property name="autoScroll">
    <bool>true</bool>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::NoEditTriggers</set>
   </property>
   <property name="tabKeyNavigation">
    <bool>false</bool>
   </property>
   <property name="showDropIndicator" stdset="0">
    <bool>true</bool>
   </property>
   <property name="dragEnabled">
    <bool>false</bool>
   </property>
   <property name="dragDropOverwriteMode">
    <bool>true</bool>
   </property>
   <property name="alternatingRowColors">
    <bool>true</bool>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::SingleSelection</enum>
   </property>
   <property name="selectionBehavior">
    <enum>QAbstractItemView::SelectRows</enum>
   </property>
   <property name="textElideMode">
    <enum>Qt::ElideRight</enum>
   </property>
   <property name="showGrid">
    <bool>true</bool>
   </property>
   <property name="gridStyle">
    <enum>Qt::SolidLine</enum>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
   <property name="cornerButtonEnabled">
    <bool>true</bool>
   </property>
   <property name="rowCount">
    <number>10</number>
   </property>
   <property name="columnCount">
    <number>4</number>
   </property>
   <attribute name="horizontalHeaderVisible">
    <bool>true</bool>
   </attribute>
   <attribute name="horizontalHeaderCascadingSectionResizes">
    <bool>false</bool>
   </attribute>
   <attribute name="horizontalHeaderHighlightSections">
    <bool>false</bool>
   </attribute>
   <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
    <bool>false</bool>
   </attribute>
   <attribute name="horizontalHeaderStretchLastSection">
    <bool>false</bool>
   </attribute>
   <attribute name="verticalHeaderVisible">
    <bool>false</bool>
   </attribute>
   <attribute name="verticalHeaderHighlightSections">
    <bool>true</bool>
   </attribute>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <column>
    <property name="text">
     <string>Código</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Apellido</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Nombre</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Fecha de nacimiento</string>
    </property>
   </column>
   <item row="0" column="0">
    <property name="text">
     <string/>
    </property>
   </item>
   <item row="0" column="1">
    <property name="text">
     <string/>
    </property>
   </item>
   <item row="0" column="2">
    <property name="text">
     <string/>
    </property>
   </item>
  </widget>
  <widget class="QLineEdit" name="tx_buscar">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>221</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Tahoma</family>
     <pointsize>11</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_apellido">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>10</y>
     <width>82</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Tahoma</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Apellido</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bt_modificar">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>420</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Verdana</family>
     <pointsize>18</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(7, 70, 124);
font: 18pt &quot;Verdana&quot;;
border-radius: 10px;
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Modificar</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="bt_nuevo">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>420</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Verdana</family>
     <pointsize>18</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(7, 70, 124);
font: 18pt &quot;Verdana&quot;;
border-radius: 10px;
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Nuevo</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="bt_enviar">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>420</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Verdana</family>
     <pointsize>18</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(7, 70, 124);
font: 18pt &quot;Verdana&quot;;
border-radius: 10px;
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Enviar QR</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>tx_buscar</tabstop>
  <tabstop>radioButton_codigo</tabstop>
  <tabstop>radioButton_apellido</tabstop>
  <tabstop>radioButton_nombre</tabstop>
  <tabstop>tableWidget</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
